flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/DataStream.scala

```diff
@@ -666,18 +666,18 @@ class DataStream[T](stream: JavaStream[T]) {
   }
 
   /**
-   * Applies the given [[ProcessFunction]] on the input stream, thereby
-   * creating a transformed output stream.
-   *
-   * The function will be called for every element in the stream and can produce
-   * zero or more output.
-   *
-   * @param processFunction The [[ProcessFunction]] that is called for each element
-   *                   in the stream.
-   */
+    * Applies the given [[ProcessFunction]] on the input stream, thereby
```

Comment:
Can you revert this formatting? I think proper java docs should be as this was before:
```
/**
 *
 */
```
instead of:
```
/**
  *
  */
```